frontend/isoko/src/App.tsx

```diff
@@ -27,6 +33,20 @@ const App: React.FC = () => {
       region: 'us-west-2',
    });
 
+   const userPool = new CognitoUserPool({
+      UserPoolId: environment.cognitoUserPoolId,
+      ClientId: environment.cognitoAppClientId,
+   });
+
+   const [isOwner, setIsOwner] = useState(false);
+   const dispatch = useAppDispatch();
+
+   useEffect(() => {
+      // checks if user is a business owner
+      const userSub = userPool.getCurrentUser().getUsername();
+      dispatch(fetchProfileAsync(userSub));
+      setIsOwner(store.getState().profile.businessOwner);
+   }, []);
```

Comment:
Correct me if I'm wrong but this code doesn't run when a user logs in. When I tested it locally when I logged in this code didn't run and I couldn't access the businessDashboard even though my account is a businessOwner